### PR TITLE
Add throwable to php try/catch block snippet

### PR DESCRIPTION
**Pull request by benjamenjohnsondev**
Add [Throwable](http://php.net/manual/en/class.throwable.php) to try catch block snippet as requested:
https://github.com/Microsoft/vscode/commit/3edd27449eaa91b3440d08e861185583d389365c#r30777567

### CODE REVIEW

**Review by oneslash — COMMENTED** [review]
Since throwable is higher interface, next catch of exception will not be reached

**Comment by benjamenhogben** [comment]
Unreachable exception removed - user editable catch parameter added, this should cover most use cases - thanks for your help @oneslash

File: extensions/php/snippets/php.snippets.json [hunk 1]
```diff
@@ -254,8 +254,8 @@
 		"body": [
 			"try {",
 			"\t${1://code...}",
-			"} catch (\\Exception \\$$e) {",
-			"\t${2://throw $$e;}",
+			"} catch (${2:\\Throwable} ${3:$$t}) {",
```
Comment (roblourens) [inline extensions/php/snippets/php.snippets.json]: Is this supposed to be `$th`?

Comment (benjamenhogben) [reply extensions/php/snippets/php.snippets.json]: Oops! Changed https://github.com/Microsoft/vscode/pull/60012/commits/1de8905bf5c0e26170aff0d7b3332de9f9f5ffe3

**Review by roblourens — APPROVED** [review]
Thanks!

**Pull request merged**